wallet/src/main/java/com/android/identity_credential/wallet/MainActivity.kt

```diff
@@ -108,50 +130,78 @@ class MainActivity : ComponentActivity() {
 
     private val provisioningViewModel: ProvisioningViewModel by viewModels()
     private val credentialInformationViewModel: CredentialInformationViewModel by viewModels()
+    private lateinit var sharedPreferences: SharedPreferences
 
     private val permissionTracker = PermissionTracker(this, mapOf(
         Manifest.permission.CAMERA to "This application requires camera permission to scan",
         Manifest.permission.NFC to "NFC permission is required to operate"
     ))
 
+    private val blePermissionTracker: PermissionTracker = if (Build.VERSION.SDK_INT >= 31) {
```

Comment:
Maybe call it `inPersonPresentationPermissionTracker` since we'd also include Wifi Aware permissions here in the (unlikely) case we'd want to support Wifi Aware as well as BLE.